WordPress/src/main/res/layout/jetpack_login_empty_view.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Minor (🔍): Consider reformatting this xml file.

PS: Same applies for the `jetpack_login_include_user_container.xml` file.